Gruntfile.js

```diff
@@ -103,14 +115,18 @@ module.exports = function(grunt) {
                     { // BOOT and EMBED
                         expand: true, cwd: 'build/',
                         src: ['boot.js', 'embed.html'],
-                        dest: 'deploy/<%= visuals.timestamp %>'
+                        dest: 'deploy/packages/interactive-brexit-companion'
                     },
                     { // ASSETS
                         expand: true, cwd: 'build/',
                         src: ['main.js', 'main.css', 'main.js.map', 'main.css.map',
                             'embed.js', 'embed.css', 'embed.js.map', 'embed.css.map',
                             'assets/**/*'],
-                        dest: 'deploy/<%= visuals.timestamp %>/<%= visuals.timestamp %>'
+                        dest: 'deploy/packages/interactive-brexit-companion/<%= visuals.timestamp %>'
+                    },
+                    {   expand: true, cwd: '.',
```

Comment:
generally you'd start the object properties on the following line, unless you were putting the whole object on one line, e.g.

```javascript
var foo = [
    {a: 10, b: 20},
    {a: 11, b: 21},
    {a: 12, b: 22}
];
```
or
```javascript
var foo = [
    {
    	a: 10,
    	b: 20
	},
    {
    	a: 11,
    	b: 21
	},
    {
    	a: 12,
    	b: 22
	}
];
```